dnsapi/dns_nodion.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env sh
+
+#
+#NODION_API_KEY="abc123"
+#
+
+NODION_API_URL="https://api.nodion.com/p/v1"
+
+########  Public functions #####################
+
+#Usage: add  _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_nodion_add() {
+  fulldomain=$1
+  txtvalue=$2
+
+  if [ -z "$NODION_API_KEY" ]; then
+    NODION_API_KEY=""
+    _err "You didn't specify a NODION_API_KEY."
+    _err "API keys can be created at https://app.nodion.com/user/security."
+    _err "Please create your key and try again."
+    return 1
+  fi
+
+  _saveaccountconf NODION_API_KEY "$NODION_API_KEY"
```

Comment:
please use `_readaccountconf_mutable ` and `_saveaccountconf_mutable `

See example:  https://github.com/acmesh-official/acme.sh/blob/master/dnsapi/dns_cf.sh#L21

Reply:
Thanks! I just pushed an update that makes use of _readaccountconf_mutable.

Reply:
_saveaccountconf_mutable